src/utils/treasury-utils.ts

```diff
@@ -109,35 +92,36 @@ export const getTreasuryDetails = async () => {
       raw_dollar: dollarValue,
     }
   })
-
   const treasuryDetails = await Promise.all(details)
   const additionalTreasuryData: TreasuryTokenType[] = []
-  const allLpTokens = [
-    ...lpTokenDetails,
-    ...convexProtocolData,
-    ...fraxLendProtocolData,
-  ]
-  allLpTokens.forEach((lp) => {
+  const allLpTokens = await Promise.all(walletDetails)
+  allLpTokens.flat().forEach((lp) => {
     if (SUPPORTED_LP_TOKENS_ADDRESSES.includes(lp.pool.id)) {
       additionalTreasuryData.push({
         id: lp.pool.adapter_id,
         contractAddress: lp.pool.controller,
         raw_dollar: Number(lp.stats.asset_usd_value),
-        token: lp.detail.supply_token_list.map((supply) => ({
-          amount: supply.amount,
-          symbol: supply.symbol,
-        })),
+        token: lp.detail.supply_token_list.map(
+          (supply: { amount: number; symbol: string }) => ({
+            amount: supply.amount,
+            symbol: supply.symbol,
+          }),
+        ),
       })
     }
   })
-
   const allTreasureDetails = [...treasuryDetails, ...additionalTreasuryData]
   const sortedTreasuryDetails = allTreasureDetails.sort(
     (a, b) => b.raw_dollar - a.raw_dollar,
   )
   let totalAccountValue = 0
   sortedTreasuryDetails.forEach((token) => {
-    totalAccountValue += token.raw_dollar
+    if (token.raw_dollar > 20000) {
```

Comment:
this should be a constant and replaced in the other side where you added the magic numbers. it doesnt take much time but makes code more easy to read and change

Reply:
Thanks for the feedback